camayoc/ui/models/pages/scans.py

```diff
@@ -91,6 +139,19 @@ def download_scan_modal(
                 self._client.driver.locator(REFRESH_BUTTON_LOCATOR).click()
         raise FailedScanException("Scan could not be downloaded")
 
+    def read_summary_modal(self) -> SummaryReportData:
+        timeout_start = time.monotonic()
+        timeout = self._client._camayoc_config.camayoc.scan_timeout
+        while timeout > (time.monotonic() - timeout_start):
```

Comment:
Sorry if this is a naive question, but could we be missing actual errors here?
Like, if the modal doesn't show up because of a frontend issue or a change in the DOM, would we just keep retrying until it times out without making it clear what went wrong?
I’m wondering if there’s a way to log or capture more info when that happens or maybe that’s not needed?